### PR TITLE
Remove mtools and block_avg dependencies

### PR DESCRIPTION
**Pull request by mattwthompson**
Addresses #2 for signac-flow 0.5

### CODE REVIEW

**Comment by mattwthompson** [comment]
I don't have cool testing set up but running the analysis script on the same data produces pretty much the same result.

image

File: README.md [hunk 1]
```diff
@@ -4,18 +4,13 @@ See [generic signac template](https://github.com/glotzerlab/signac-project-templ
...
 # Installation
 
-Install these packages:
-
-* [block_avg](https://github.com/tcmoore3/block_avg)
-* [mtools](https://github.com/mattwthompson/mtools)
-
-Install other dependencies:
+First, verify that each package in `requirements.txt` is installed. If not
+already installed, They should all be available via `pip`, `conda`, etc.
```
Comment (summeraz) [inline README.md:8]: Might want to mention the additional conda channels that are required. `mosdef` for `mbuild` and either `glotzer` or `conda-forge` for `signac-flow`.

**Comment by summeraz** [comment]
One minor comment, otherwise looks good to me!

**Comment by mattwthompson** [comment]
Thanks @summeraz!

**Pull request merged**